siriuspy/siriuspy/clientconfigdb/pvsconfig.py

```diff
@@ -0,0 +1,128 @@
+"""PVs Configs Handler."""
+
+import time as _time
+from epics import get_pv as _get_pv
+import numpy as _np
+from siriuspy.clientconfigdb import ConfigDBDocument as _ConfigDBDocument
+
+
+_TIMEOUT = 0.5
+
+
+class PVsConfig(_ConfigDBDocument):
+    """Class to handle PVs configurations."""
+
+    PVs = dict()
+
+    def __init__(self, config_type='global_config', name=None, url=None):
+        """Init."""
+        super().__init__(config_type, name=name, url=url)
+
+    def read(self, timeout=_TIMEOUT):
+        """Read machine state."""
+        new_config_value = dict()
+        new_config_value['pvs'] = list()
+
+        # connect
+        template = self.get_value_template()
```

Comment:
Instead of looping here, why not create a `connect` method? It would also be nice to have a `connected` property.